### DISCUSSION TITLE
Using a cloud server

### DISCUSSION THREAD

**Discussion by mojtababouraghi** [MRColorR/money4band#38] [Q&A]
Hello
Is it possible to use this program on a cloud server like Hetzner cloud servers?

**Comment by MRColorR** [comment]
hi, it should work on any linux/windows/Mac system that can run docker. 
In your case to run this project on a Hetzner Cloud server, you would need to do the following:

1) Create a Hetzner Cloud server with a suitable Linux distribution (e.g., Ubuntu, Debian, or CentOS). Ensure that the server has enough resources (CPU, RAM, and disk space) to run your project.

2) Access the server via SSH and update the package lists and installed packages.

3) Install Docker and docker-compose on the server. You can follow the official Docker installation documentation for the appropriate Linux distribution. 

4) Clone money4band GitHub repository to the server or download the latest release.

5) follow the readme instructions to run and setup the project.

Keep in mind that you might need to adjust the project configuration, such as exposed ports or environment variables, depending on your specific requirements and the server setup. Additionally, you might need to open ports on the server's firewall 

Feel free to reopen the discussion if you need more details or if you'd like to add more useful info for other users.

**Reply by mojtababouraghi** [reply]
Thank you for your answer
But according to the rules mentioned on [this page](https://support.honeygain.com/hc/en-us/articles/360011078760-Error-Unusable-network),
**We are not allowed to use Data Center (DCH), (ORG) Organization, and Reserved (RSV) IP types**
Is there a solution for this issue?

**Reply by MRColorR** [reply]
Hi, unfortunately some apps have more restricted ToS and the user should respect it. This is also stated in the readme and in the wiki. 
In the next money4band release, more advanced functions will be released but also a compatibility matrix will appear. This way you will be able to select which app run in which environment.
